src/utils/ContractUtils.ts

```diff
@@ -101,3 +101,9 @@ export const getCampaignKey = (projectToken: string) => {
   const hash = ethers.utils.keccak256(encodedParams);
   return hash;
 };
+
+export const checkLinkValidity = (link: string): boolean => {
+  const pattern =
```

Comment:
Is it possible to use this? https://gabrieleromanato.name/nodejs-check-the-validity-of-a-website-ssl-certificate